saagieapi/jobs/jobs.py

```diff
@@ -1627,3 +1823,107 @@ def count_instances_by_status(self, job_id):
         return self.saagie_api.client.execute(
             query=gql(GQL_COUNT_INSTANCES_BY_SELECTOR), variable_values={"jobId": job_id}
         )
+
+    def count_instances_by_date(self, job_id: str, date_before: str):
+        """Count job instances by date
```

Comment:
Maybe adjust the description to: `Count removable job instance by date`

Reply:
This function counts all instances of a job by date, not only the removable instances. So I think the description is correct.

Reply:
I changed the description and also the name of the function to : count_deletable_instances_by_date